src/protocols/secure_channel/CASESession.h

```diff
@@ -101,13 +105,14 @@ class DLL_EXPORT CASESession
      * @param myNodeId                      Node id of local node
      * @param peerNodeId                    Node id of the peer node
      * @param myKeyId                       Key ID to be assigned to the secure session on the peer node
+     * @param exchangeCtxt                  The exchange context to send and receive messages with the peer
      * @param delegate                      Callback object
      *
      * @return CHIP_ERROR      The result of initialization
      */
     CHIP_ERROR EstablishSession(const Transport::PeerAddress peerAddress, OperationalCredentialSet * operationalCredentialSet,
                                 Optional<NodeId> myNodeId, NodeId peerNodeId, uint16_t myKeyId,
-                                SessionEstablishmentDelegate * delegate);
+                                Messaging::ExchangeContext * exchangeCtxt, SessionEstablishmentDelegate * delegate);
```

Comment:
Why do we pass `myNodeId` when we are already passing `operationalCredentialSet`?  This seems inherently redundant.  What happens if `myNodeId` is different than the authenticated RDN in `operationalCredentialSet`?